src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/CrazyPriceHost.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Exadel.CrazyPrice.Common.Configurations;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.IO;
+
+namespace Exadel.CrazyPrice.Common
+{
+    public static class CrazyPriceHost
+    {
+        public static void CheckLoggerAndStartHost(IHostBuilder hostBuilder,
+            string initMessage, string errorInitMessage)
+        {
+            const string useLogging = "Logging:UseLogging";
+            const string loggerName = "Logging:LoggerName";
+            const string nLog = "nlog";
+            const string serilog = "serilog";
+            const string appsettings = "appsettings.json";
+
+            var configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile(appsettings)
+                .Build();
+
+            if (!bool.TryParse(configuration.GetSection(useLogging).Value, out var hasLogging))
```

Comment:
Create methods for reading configuration parameters which will raise exception if no required data found. Also create class for Serilog/Nlog configuration and fill them here and pass to each logger own class

Reply:
Removed.